marconi-mamba/src/Marconi/Mamba/Api/Types.hs

```diff
@@ -60,8 +61,7 @@ data MambaEnv = MambaEnv
 makeClassy ''MambaEnv
 
 data QueryExceptions
-    = AddressNotInListError QueryExceptions
-    | AddressConversionError QueryExceptions
+    = AddressConversionError QueryExceptions
     | TxRefConversionError QueryExceptions
```

Comment:
this one is also no longer needed, my bad as I should've taken out when we switched to `unspent_transaction` table